app/controllers/items_controller.rb

```diff
@@ -1,5 +1,6 @@
-class ItemsController < ApplicationController
+require 'open-uri'
```

Comment:
Oooooooh what is this?!

Reply:
It's a ruby module used to analyze urls. For example, you can go like `open("www.google.com).status` and it will return the status code and message `["200", "OK"]`.

I didn't dig too much into it's functionality but it did the trick to fix the bug I was facing.